src/WaveTrack.cpp

```diff
@@ -48,6 +48,7 @@ from the project that will own the track.
 
 #include "Project.h"
 #include "ProjectRate.h"
+#include "Clipboard.h"
```

Comment:
Although it doesn’t make a cycle, this dependency on Clipboard feels wrong to me.